app/screens/ScheduleScreen/ScheduleDayPicker.tsx

```diff
@@ -9,8 +9,6 @@ import Animated, {
 } from "react-native-reanimated"
 import { colors, spacing, typography } from "../../theme"
 import { reportCrash } from "../../utils/crashReporting"
-import { parseDate } from "../../utils/formatDate"
-import { Schedule } from "./ScheduleScreen"
```

Comment:
Things this component no longer needs to care about. This also makes this component project independent. I could rename this component `DayPickerSlider`.